go/cmd/dolt/commands/batsee.go

```diff
@@ -67,13 +66,21 @@ func (b BatseeCmd) Hidden() bool {
 	return true
 }
 
+const (
+	threadsF  = "threads"
```

Comment:
I think we should just say "threadsFlag" instead of "threadsF". There's no reason to abbreviate it.

Reply:
> Regarding making is a subcommand - No strong reason other than having everything in one executable. I thought the signal handling would come in handy, but it's really only a few lines of code to replicate that. More than anything else it was easiest.

I have a strong reason no to do this - Windows. The signal handling won't work. I'll make it a separate command.

Reply:
I disagree, but don't care enough to argue.